Assignment/Program.cs

```diff
@@ -1,7 +1,21 @@
 ﻿namespace Assignment {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+         Console.Write ("Enter total votes : ");
+         string input = Console.ReadLine ()!.ToLower ().Replace (" ", "");
+         VotingContext (input);
+      }
+
+      public static void VotingContext (string input) {
+         Dictionary<char, int> vote = new ();

```

Comment:
Dictionary is not guaranteed that the order will be reserved. Re write using a list and tuple.

Reply:
Re written using list